var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -32,7 +32,9 @@ class Flex(AutotoolsPackage):
     homepage = "https://github.com/westes/flex"
     url = "https://github.com/westes/flex/releases/download/v2.6.1/flex-2.6.1.tar.gz"
 
-    version('2.6.4', '2882e3179748cc9f9c23ec593d6adc8d')
+    # Avoid flex '2.6.4' because it fails compilation with GCC 7.2.0 Linux x64.
+    # See: https://github.com/westes/flex/issues/219
+    # version('2.6.4', '2882e3179748cc9f9c23ec593d6adc8d')
```

Comment:
I would make this change in another PR. I don't think we should remove this version altogether. Let's decide on something in #6812 and leave it for another PR.